File: Beecrowd/3302 Resposta Certa/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int n, r;

        n = sc.nextInt();

        for (int i = 1; i <= n; i++) {
            r = sc.nextInt();
            System.out.println("resposta " + i + ": " + r);
        }

        sc.close();
    }

}

File: Beecrowd/3302 Resposta Certa/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    int n, r;

    cin >> n;

    for (int i = 1; i <= n; i++) {
        cin >> r;
        cout << "resposta " << i << ": " << r << endl; 
    }
 
    return 0;
}